tests/release/pipelines/push_disk_images_to_cdn.go

```diff
@@ -0,0 +1,276 @@
+	err = managedFw.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(managedNamespace, releasecommon.PublicSecretNameAuth, rhaiServiceAccountName, true)
+	Expect(err).ToNot(HaveOccurred())
+
+	_, _ = managedFw.AsKubeAdmin.CommonController.CreateRole("role-release-service-account", managedNamespace, map[string][]string{
+		"apiGroupsList": {""},
+		"roleResources": {"secrets"},
+		"roleVerbs":     {"get", "list", "watch"},
+	})
+
+	_, _ = managedFw.AsKubeAdmin.CommonController.CreateRoleBinding("role-release-service-account-binding", managedNamespace, "ServiceAccount", "release-service-account", managedNamespace, "Role", "role-release-service-account", "rbac.authorization.k8s.io")
+
+	defaultEcPolicySpec := ecp.EnterpriseContractPolicySpec{
+		Description: "Red Hat's enterprise requirements",
+		PublicKey:   fmt.Sprintf("k8s://%s/%s", managedNamespace, releasecommon.PublicSecretNameAuth),
+		Sources: []ecp.Source{{
+			Name:   "Default",
+			Policy: []string{releasecommon.EcPolicyLibPath, releasecommon.EcPolicyReleasePath},
+			Data:   []string{releasecommon.EcPolicyDataBundle, releasecommon.EcPolicyDataPath},
+		}},
+		Configuration: &ecp.EnterpriseContractPolicyConfiguration{
+			Exclude: []string{"step_image_registries", "tasks.required_tasks_found:prefetch-dependencies"},
+			Include: []string{"@slsa3"},
+		},
+	}
+
+	_, err := managedFw.AsKubeDeveloper.TektonController.CreateEnterpriseContractPolicy(rhaiECPName, managedNamespace, defaultEcPolicySpec)
+	Expect(err).NotTo(HaveOccurred())
+
+}
+
+func createRHAIReleasePlanAdmission(rhaiRPAName string, managedFw framework.Framework, devNamespace, managedNamespace, rhaiAppName, rhaiECPName, pathInRepoValue string) {
+	var err error
+
+	data, err := json.Marshal(map[string]interface{}{
+		"mapping": map[string]interface{}{
+			"components": []map[string]interface{}{
+				{
+					"name": rhaiComponentName,
+					"staged": map[string]interface{}{
+						"destination": "rhelai-1_DOT_1-for-rhel-9-x86_64-isos",
+						"version": "1.1",
+						"files": []map[string]interface{}{
+							{
+								"filename": "rhel-ai-nvidia-1.1-{{ timestamp }}"+util.GenerateRandomString(4)+"-x86_64-kvm.qcow2",
+								"source": "disk.qcow2",
+							},
+							{
+								"filename": "rhel-ai-nvidia-1.1"+"-{{ timestamp }}"+util.GenerateRandomString(4)+"-x86_64.raw",
+								"source": "disk.raw",
+							},
+							{
+								"filename": "rhel-ai-nvidia-1.1"+"-{{ timestamp }}"+util.GenerateRandomString(4)+"-x86_64-boot.iso",
+								"source": "install.iso",
+							},
+						},
+					},
+					"contentGateway": map[string]interface{}{
+						"productName": "E2ETest Red Hat Enterprise Linux AI",
+						"productCode": "RHELAIE2ETest",
+						"productVersionName": "RHELAI 1.1",
+						"filePrefix": "rhel-ai-nvidia-1.1",
+					},
+				},
+			},
+		},
+		"tags": []string{"time-{{ timestamp }}", "git-{{ git_sha }}" },
+		"cdn": map[string]interface{}{
+			"env": "stage",
```

Comment:
I was told in my testing to use the qa env, not stage. Are we sure we want stage?

Reply:
What did you set for the "destination" in "qa" env? It seems need to be created before the test.  I always met error when I used "qa" env.

Reply:
`rhel-7-server-for-power-le-rhscl-isos__7_DOT_1__ppc64le`

Reply:
Thanks